src/libraries/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.Directory.Create.cs

```diff
@@ -49,15 +48,15 @@ public void FileSystemWatcher_Directory_Create()
         [Fact]
         public void FileSystemWatcher_Directory_Create_InNestedDirectory()
         {
-            using (var dir = new TempDirectory(GetTestFilePath()))
-            using (var firstDir = new TempDirectory(Path.Combine(dir.Path, "dir1")))
-            using (var nestedDir = new TempDirectory(Path.Combine(firstDir.Path, "nested")))
-            using (var watcher = new FileSystemWatcher(dir.Path, "*"))
+            string firstDir = Path.Combine(TestDirectory, "dir1");
+            string nestedDir = Path.Combine(firstDir, "nested");
```

Comment:
```suggestion
            string nestedDir = Path.Combine(firstDir, "dir1", "nested");
```
and others below similarly